frontend/ui/widget/dictquicklookup.lua

```diff
@@ -865,7 +867,8 @@ function DictQuickLookup:update()
     if self.is_html and self.shw_widget then
         -- Reuse our ScrollHtmlWidget (self.shw_widget)
         -- NOTE: The recursive free via our WidgetContainer (self[1]) above already released the previous MµPDF document instance ;)
-        self.text_widget.htmlbox_widget:setContent(self.definition, self:getHtmlDictionaryCss(), Screen:scaleBySize(self.dict_font_size))
+        local html_resource_directory = util.splitFilePathName(self.dictionary_file)
```

Comment:
Same as above. It might be better to store the dictionary directory instead of the ifo file in `tidyMarkup`.

Somewhere around here:
```lua
            if ifo.fix_html_func then
                local dict_path = util.splitFilePathName(ifo.file)
```